cluster/examples/kubernetes/ceph/kube-registry-psp.yaml

```diff
@@ -0,0 +1,67 @@
+# Example showing how to run the kube-registry application pod security policies.
+# Attempts have been made to make sure the policy, role, and binding in this example follow
+# best practices and are a minimal set of permissions; however, this is not a guarantee of a
+# minimal policy config.
+# Because the kube-registry service account is in the kube-registry.yaml file, that must be created
+# first or at the same time as this file.
+---
+# Create a pod security policy (psp) for the service account
+# that is as minimal as possible.
+# TODO: This could probably be more minimal
```

Comment:
you might remove this TODO since it was mentioned above that it might not be minimal